src/main/scala/io/phdata/retirementage/loadgen/KuduLoadGenerator.scala

```diff
@@ -0,0 +1,122 @@
+package io.phdata.retirementage.loadgen
+
+import io.phdata.retirementage.domain.GlobalConfig
+import io.phdata.retirementage.loadgen.LoadGenerator.{generateTable, generateTableFromParent}
+import org.apache.kudu.client.CreateTableOptions
+import org.apache.spark.sql.types.{LongType, StringType, StructField, StructType}
+import org.apache.spark.sql._
+import org.apache.spark.storage.StorageLevel
+import org.apache.kudu.spark.kudu._
+import scala.collection.JavaConverters._
+
+object KuduLoadGenerator {
+  val defaultSchema = StructType(StructField("id", StringType, false) :: Nil)
+    .add(StructField("payload", StringType, false))
+    .add(StructField("dimension_id", StringType, false))
+    .add(StructField("expiration_date", StringType, false))
+  val defaultKey = Seq("id")
+
+  def main(args: Array[String]): Unit = {
```

Comment:
This shouldn't have its own main method or cli arguments, you can work it into the other main method, see comment in LoadGeneratorConfig.scala